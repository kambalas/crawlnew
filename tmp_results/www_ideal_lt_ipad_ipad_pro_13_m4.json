[
  {
    "name": "iPad Pro 13\" Wi-Fi 256GB Standard Glass",
    "price": "1 579,00 €",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi 512GB Standard Glass",
    "price": "1 829,00 €",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi 1TB Standard Glass",
    "price": "2 309,00 €",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi 2TB Standard Glass",
    "price": "2 789,00 €",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi + Cellular 256GB Standard Glass",
    "price": "1 829,00 €",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi + Cellular 512GB Standard Glass",
    "price": "2 079,00 €",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi + Cellular 1TB Standard Glass",
    "price": "2 559,00 €",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi + Cellular 2TB Standard Glass",
    "price": "3 039,00 €",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi 1TB Nano-Texture Glass",
    "price": "2 439,00 €",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi 2TB Nano-Texture Glass",
    "price": "2 919,00 €",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi + Cellular 1TB Nano-Texture Glass",
    "price": "2 689,00 €",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi + Cellular 2TB Nano-Texture Glass",
    "price": "3 169,00 €",
    "error": false
  }
]