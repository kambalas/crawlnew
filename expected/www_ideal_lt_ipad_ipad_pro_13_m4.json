[
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  },
  {
    "name": "iPad Pro 13\" Wi-Fi 256GB Standard Glass",
    "price": "1579",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi 512GB Standard Glass",
    "price": "1829",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi 1TB Standard Glass",
    "price": "2309",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi 2TB Standard Glass",
    "price": "2789",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi + Cellular 256GB Standard Glass",
    "price": "1829",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi + Cellular 512GB Standard Glass",
    "price": "2079",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi + Cellular 1TB Standard Glass",
    "price": "2559",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi + Cellular 2TB Standard Glass",
    "price": "3039",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi 1TB Nano-Texture Glass",
    "price": "2439",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi 2TB Nano-Texture Glass",
    "price": "2919",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi + Cellular 1TB Nano-Texture Glass",
    "price": "2689",
    "error": false
  },
  {
    "name": "iPad Pro 13\" Wi-Fi + Cellular 2TB Nano-Texture Glass",
    "price": "3169",
    "error": false
  }
]